Branch-SDK/Branch-SDK/Requests/BranchOpenRequest.m

```diff
@@ -82,6 +82,7 @@ - (void)makeRequest:(BNCServerInterface *)serverInterface key:(NSString *)key ca
 
 - (void)processResponse:(BNCServerResponse *)response error:(NSError *)error {
     if (error) {
+        [BranchOpenRequest releaseOpenResponseLock];    
```

Comment:
Won't the call to self.callback(NO, error) on line 88 passthrough that network error? I believe it will but you should confirm while testing with no network & network attenuation